src/components/asset-panel/selector.css

```diff
@@ -56,7 +56,7 @@ $fade-out-distance: 100px;
 
 .list-item {
     width: 5rem;
-    min-height: 5rem;
+    height: 5rem;
```

Comment:
should I be worried about this change from `min-height` to `height`? Was there a particular reason we had min-height before?